_data/projects/parameter-framework.yml

```diff
@@ -0,0 +1,8 @@
+name: Parameter Framework
+desc: Plugin-based and rule-based framework for describing and controlling parameters
+site: https://01.org/parameter-framework/
+tags:
+- C++
+upforgrabs:
+  name: trivial
+  link: https://github.com/01org/parameter-framework/issues
```

Comment:
Could you set this link to `https://github.com/01org/parameter-framework/labels/trivial` instead of the displaying the entire list of issues?